e2e/pollers.go

```diff
@@ -28,17 +27,17 @@ const (
 	execOnPodPollingTimeout                = 2 * time.Minute
 	extractClusterParametersPollingTimeout = 3 * time.Minute
 	extractVMLogsPollingTimeout            = 5 * time.Minute
-	waitUntilNodeReadyPollingTimeout       = 3 * time.Minute
+	waitUntilNodeReadyPollingTimeout       = 6 * time.Minute
```

Comment:
I'm thinking about removing all these timeouts and set a single timeout per test.
Large number of timeouts are more annoying to tweak.